tests/functional/issues-auth.js

```diff
@@ -53,7 +53,28 @@ define(
           .then(function(text) {
             assert.equal(text, "Close Issue", "Button says Close not Reopen");
           });
+      },
+
+      "Pressing 'l' opens the label editor box": function () {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/70"),
+          ".wc-Issue-commentSubmit",
+          true
```

Comment:
oh sorry, i forgot to remove that.
i was just testing to see if the timeout was the problem with the tests that are intermittently not working.
i'll change it back.